unity-client/Assets/Scripts/MainScripts/DCL/ScriptableObject/CommonScriptableObjects.cs

```diff
@@ -122,6 +122,10 @@ public static class CommonScriptableObjects
     public static RendererState rendererState => GetOrLoad(ref rendererStateValue, "ScriptableObjects/RendererState");
     private static RendererState rendererStateValue;
 
+    public static BooleanVariable focusState => GetOrLoad(ref focusStateValue, "ScriptableObjects/FocusState");
+    private static BooleanVariable focusStateValue;
```

Comment:
`= true`?

Reply:
Just added the initialization here:
https://github.com/decentraland/explorer/blob/d1b391fafbdb2ff2c04f1e2f426a611167d2ccd8/unity-client/Assets/Scripts/MainScripts/DCL/Bridges/FocusStateBridge/FocusStateBridge.cs#L9